test/tests/jetty-hello-web/run.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+set -eo pipefail
+
+dir="$(dirname "$(readlink -f "$BASH_SOURCE")")"
+
+image="$1"
+
+# Pull a client image with curl for testing
+clientImage='appropriate/curl'
```

Comment:
Why not just `buildpack-deps:jessie-curl`, it is what `java:jre` images are based on, so it will already exist :wink:.